crates/swc_ecma_transforms_compat/tests/es2015_classes.rs

```diff
@@ -7008,14 +6998,13 @@ let A = function(B) {
   _inherits(A, B);
   var _super = _createSuper(A);
   function A() {
-    _classCallCheck(this, A);
-    var _super_foo, _baz, _super_baz, _quz;
-    var _this;
-    _this.foo = (_super_foo = +B.prototype.foo) + 1, _super_foo;
-    _this.bar = B.prototype.bar + 123;
```

Comment:
Babel's behavior is wrong.

Reply:
```JavaScript 
class B {
  _foo = 1;
  get foo() {
    return this._foo;
  }
  set foo(v) {
    this._foo = v;
  }
}

class A extends B {
  bar() {
    super.foo++;
  }
}

const a1 = new A();
const a2 = new A();

console.log(a1.foo, a2.foo); // 1 1

a1.bar();
console.log(a1.foo, a2.foo); // 2 1

a1.bar();
console.log(a1.foo, a2.foo); // 3 1
```

Reply:
[Babel](https://babeljs.io/repl/#?browsers=chrome%2045&build=&builtIns=false&corejs=3.21&spec=false&loose=false&code_lz=MYGwhgzhAEBC0G8BQ1oH0BmB7L0C80AjANwrQDmApgC7TZYAUAlImagE40Cu7AdtNQAWASwgA6TDlKoAvmQg06OBgDcWyVKiGiJ9fNBXTocuUlCQYAQWiUAHtUq8AJjHgboAIzDtmrTdAguAAdKdjF6AGoIoxMkMyxeCFowQn1eSgB3aEtmUmAEpOgwACY0zOzcuPzErBBKMRAscgYU8JwAGiLitqwmYmgAegGiIjjWrx8--Jq6hqaWwh7Okp6-weHSwjHFicrqiFr6xubW-mXu-jWh6ABmIiA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=false&presets=env&prettier=false&targets=&version=7.18.12&externalPlugins=&assumptions=%7B%22constantSuper%22%3Atrue%7D)'s result 
```
1 1
NaN 1
NaN 1
```